src/ngx_http_lua_ssl.c

```diff
@@ -34,4 +34,35 @@ ngx_http_lua_ssl_init(ngx_log_t *log)
 }
 
 
+int
+ngx_http_lua_ssl_password_callback(char *buf, int size, int rwflag,
```

Comment:
maybe `ngx_http_lua_ssl_password_callback` is a useful function ? I mean that it will be used in other scene like `cosocket` in future.